CONTRIBUTING.md

```diff
@@ -6,6 +6,14 @@ The concord-bft project team welcomes contributions from the community. If you w
 signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any
 questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq).
 
+## Important
+
+Concord-bft is used in multiple commercial projects which are naturally prioritized by the concord team.
+
+Sometimes there are delays until code and documentation is merged into this repository, mainly because of the need to strip proprietary information from the customers systems.
```

Comment:
I think it is expected that any open source project takes some time to process incoming PRs, regardless of whether its business model. So, this seems kind of obvious, and IMHO, it needn't be stated explicitly.

Reply:
As above, this is based on https://github.com/vmware/concord-bft/issues/426#issuecomment-594690171, to clarify that some constructs can evolve within commercial products ("paying customers", i just avoided this terminology).

This does not affect PRs, but the VMware internal process ("strip proprietary information")

Reply:
updated a bit.